### PR TITLE
Update #48

### PR DESCRIPTION
**Pull request by rpitingolo**
@NeighborhoodInfoDC/review I edited the %Parcel_geo_update_new macro to remove legacy code and add the finalize macro.

### CODE REVIEW

File: Prog/Parcel_geo_ownerpt_2018_05.sas [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+%let update_date = 2018_05;
+%let revisions = Updated through 2018-05;
+
+
+/** Don't need to edit this code **/
+
+%let update_file = Ownerpt_&update_date.;
+
+%Parcel_geo_update_new( update_file=&update_file, finalize=&finalize, keep_vars =
```
Comment (lhendey) [inline Prog/Parcel_geo_ownerpt_2018_05.sas]: Gettting and error on line 33. WARNING: Apparent symbolic reference FINALIZE not resolved.
ERROR: The text expression &FINALIZE contains a recursive reference to the macro variable FINALIZE.  The macro variable 
       will be assigned the null value.

**Comment by rpitingolo** [comment]
@lhendey I see the error. When you say N=0 you mean the dataset Parcel_geo_ownerpt_2018_05 has no data?

**Comment by lhendey** [comment]
@rpitingolo - yep - it produces a proc contents with vars but no observations

**Comment by lhendey** [comment]
@rpitingolo - yes works now. apparently MAR spelled  GUETHLER'S WAY SE wrong. Left out the H and the 's.

**Pull request merged**